lib/gradient/error.ex

```diff
@@ -401,6 +401,10 @@ defmodule Gradient.Error do
     line(error) == line
   end
 
+  defp ignore?({:regex, regex}, {file, _error}, _kind) do
+    Regex.match?(regex, to_string(file))
+  end
+
   defp ignore?(_ignore, _file_error, _kind), do: false
```

Comment:
If someone defines an invalid line in gradient_ignore it would be a silent failure.

Would you like me to open a PR to raise with a clear message around what lines can be added to gradient ignore?

Reply:
That would be a great improvement 👍